File: admin/src/types/generalCertificate.ts
export interface GeneralCertificateResponse {
  generalCertificateId: number;
  title: string;
  description: string | null;
  image: string;
  displayOrder: number;
  isActive: boolean;
  createdAt: string;
  createdBy: string | null;
  updatedAt: string | null;
  updatedBy: string | null;
}

export interface CreateGeneralCertificateRequest {
  title: string;
  description: string | null;
  image: string | null;
  displayOrder: number;
  isActive: boolean;
}

export interface UpdateGeneralCertificateRequest {
  generalCertificateId: number;
  title: string;
  description: string | null;
  image: string | null;
  existingImage: string | null;
  displayOrder: number;
  isActive: boolean;
}

File: backend/src/MalihaPolytex.Domain/Entities/GeneralCertificate.cs
namespace MalihaPolytex.Domain.Entities;

public class GeneralCertificate
{
    public int GeneralCertificateId { get; set; }

    public string Title { get; set; } = string.Empty;

    public string? Description { get; set; }

    public string Image { get; set; } = string.Empty;

    public int DisplayOrder { get; set; }

    public string IsActive { get; set; }

    public DateTime CreatedAt { get; set; }

    public string? CreatedBy { get; set; }

    public DateTime? UpdatedAt { get; set; }

    public string? UpdatedBy { get; set; }
}